packages/services/ds/src/workflows/server/pegboard/mod.rs

```diff
@@ -391,6 +391,14 @@ async fn setup(
 			owner: pp::ActorOwner::DynamicServer {
 				server_id: input.server_id,
 			},
+			metadata: util::serde::Raw::new(&json!({
```

Comment:
let's make this stick to the schema of the actors api to reduce confusion.

it looks like it's complicated to fully populate the metadata bc the pb client doesn't have everything. ideally we populate as much as we can to match the api schema.

Reply:
It doesnt make sense to populate the entire config because it contains things like env vars and args which are already included elsewhere. I just did a small portion of the config, what else did you have in mind?

Reply:
oh i see youre not referring to the input but the Actor struct returned by the get endpoint

Reply:
sgtm. realizing it might not really make sense to do that anyways. let's structure it a bit more so we can add more common properties that servers will need in the future –

```typescript
{
actor: { id, tags },
id,
environment: { id },
cluster: { id },
// leave out datacenter for now
region: { id },
build: { id, tags }  // if you don't have build tags, we'll add it later
}
```

these props need to be camel case for js.

Reply:
The props are converted to camel case by the js glue. Metadata isnt specific to isolates

Reply:
Why not leave datacenter? more info the better right?

Reply:
we haven’t stabilized the dc api yet, so we should avoid from exposing anything until it’s ready